composer.json

```diff
@@ -40,13 +40,13 @@
         "psalm/plugin-phpunit": "^0.16",
         "psalm/plugin-symfony": "^3.0",
         "rector/rector": "^0.12",
-        "symfony/browser-kit": "^4.4 || ^5.3 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.3 || ^6.0",
-        "symfony/http-foundation": "^4.4 || ^5.3 || ^6.0",
+        "symfony/browser-kit": "^5.4 || ^6.0",
+        "symfony/dependency-injection": "^5.4 || ^6.0",
+        "symfony/framework-bundle": "^5.4 || ^6.0",
+        "symfony/http-foundation": "^5.4 || ^6.0",
         "symfony/http-kernel": "^4.4.13 || ^5.3 || ^6.0",
         "symfony/phpunit-bridge": "^6.0",
-        "symfony/twig-bundle": "^4.4 || ^5.3 || ^6.0",
+        "symfony/twig-bundle": "^5.4 || ^6.0",
         "vimeo/psalm": "^4.7.2"
     },
     "conflict": {
```

Comment:
Should we remove this conflict? Sonata 3 wont be compatible with this anyway

Reply:
Done